common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -273,7 +273,9 @@ function () {
                         );
                     }
 
-                    el.trigger('html5:' + eventName, arguments);
+                    var vent = document.createEvent('Event');
```

Comment:
I'm not sure `createEvent` is compatible with Internet Explorer.
see: https://developer.mozilla.org/en-US/docs/Web/API/Document/createEvent

we need to test this manually, as our automation doesn't run against IE.